src/middleware/queries.ts

```diff
@@ -1703,3 +1704,65 @@ export function transactionHistoryByConfidentialAssetQuery(
     variables: { size: size?.toNumber(), start: start?.toNumber(), assetId },
   };
 }
+
+export type ConfidentialTransactionsByConfidentialAccountArgs = {
+  accountId: string;
+  direction: 'Incoming' | 'Outgoing' | 'All';
```

Comment:
Should this be an enum for consistency? Not sure why this should be different than status right below it.

Reply:
i guess string is alright then